src/experimental/effects/reflection-effect/reflection-effect-fragment.glsl

```diff
@@ -18,9 +18,6 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
 
-/* fragment shader for the grid-layer */
-#define SHADER_NAME reflection-effect-fs
```

Comment:
Why are we removing the SHADER_NAMEs? We will still parse these (don't need glslify for that).